src/Workleap.Extensions.Http.Authentication.ClientCredentialsGrant/ClientCredentialsTokenManagementService.cs

```diff
@@ -17,16 +18,28 @@ internal class ClientCredentialsTokenManagementService : IClientCredentialsToken
     private readonly ConcurrentDictionary<string, Lazy<Task<ClientCredentialsToken>>> _lazyGetNewTokenTasks;
     private readonly IClientCredentialsTokenEndpointService _tokenEndpointService;
     private readonly IClientCredentialsTokenCache _tokenCache;
+    private readonly CancellationToken _backgroundRefreshCancellationToken;
 
-    public ClientCredentialsTokenManagementService(IClientCredentialsTokenEndpointService tokenEndpointService, IClientCredentialsTokenCache tokenCache)
+    private long _backgroundRefreshedTokenCount;
```

Comment:
If this is only for testing purposes, I think an int or uint would be enough

Reply:
Even though we only validate the value in tests, it's still going to be incremented in production environments. Very unlikely that we reach int.MaxValue or uint.MaxValue, but still.